packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -355,11 +356,12 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
       isSmall ?
         (
           <Panel
-            className={ css('ms-Dropdown-panel', styles.panel) }
+            className={ css('ms-Dropdown-panel', styles.panel, panelProps ? panelProps.className : undefined) }
```

Comment:
!!panelProps && panelProps.className instead of a ternary should work fine.

Reply:
Hi, 
I followed the style implied on line 378:
`className={ css('ms-Dropdown-callout', styles.callout, calloutProps ? calloutProps.className : undefined) }`

Functionally its the same thing, let me know if you want me to change both.

Reply:
I'd say we're moving away from the ternaries unless the 3rd option is actually valuable.

Reply:
Cool. I will update both then. What about the other comment? any other concerns before I update?

Reply:
Looks great to me. I approved